### PR TITLE
Complete README update

### PR DESCRIPTION
**Pull request by rubymorillo**
Hey team,

Per ongoing convos with @indirect, we've revamped the README. I rewrote content and rearranged various sections.

Thanks so much for the contribution!
To make reviewing this PR a bit easier, please fill out answers to the following questions.

### What was the end-user problem that led to this PR?

The problem was...we wanted to make an easy-to-read README that included clear instructions for contributing to Bundler as well as providing several troubleshooting options up front.

### What was your diagnosis of the problem?

My diagnosis was...to rewrite the things and rearrange sections!

### What is your fix for the problem, implemented in this PR?

My fix...was to make copy and layout changes.

### Why did you choose this fix out of the possible options?

I chose this fix because...@indirect and I discussed it beforehand :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,21 @@ Bundler makes sure Ruby applications run the same code on every machine.
 
 It does this by managing the gems that the application depends on. Given a list of gems, it can automatically download and install those gems, as well as any other gems needed by the gems that are listed. Before installing gems, it checks the versions of every gem to make sure that they are compatible, and can all be loaded at the same time. After the gems have been installed, Bundler can help you update some or all of them when new versions become available. Finally, it records the exact versions that have been installed, so that others can install the exact same gems.
 
+To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md).
+
 ### Installation and usage
 
-To install (or update to the latest version):
+[Visit the Bundler site](http://bundler.io) to access the complete documentation.
```
Comment (segiddins) [inline README.md]: I think this line of instructions should be "above the fold", since its not specific to installation & usage

File: README.md [hunk 2]
```diff
@@ -29,30 +35,31 @@ bundle install
...
+### Contributing
 
-To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md).
+While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together. So if you'd like to contribute to Bundler, that's awesome, and we <3 you!
 
-To get in touch with the Bundler core team and other Bundler users, please see [getting help](doc/contributing/GETTING_HELP.md).
+We have a [guide with recommended first steps](doc/contributing/README.md) that we suggest anyone interested in contributing to Bundler review first. Once you’ve completed those steps, feel free to get started contributing in any of the following ways:
 
-### Contributing
+- Adding new sections or making edits to the [documentation website](http://bundler.io) and [man pages](http://bundler.io/man/bundle.1.html)
```
Comment (segiddins) [inline README.md]: should these instead link to the repos where people can contribute?

File: README.md [hunk 3]
```diff
@@ -29,30 +35,31 @@ bundle install
...
+- [Reviewing pull requests](https://github.com/bundler/bundler/pulls)
+- [Backfilling unit tests](https://github.com/bundler/bundler/tree/master/spec/bundler) for modules that [lack coverage](https://codeclimate.com/github/bundler/bundler/coverage)
 
-If you'd like to contribute to Bundler, that's awesome, and we <3 you. There's a guide to contributing to Bundler (both code and general help) over in [our documentation section](doc/README.md).
+### Get support
 
-While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together.
+- **Troubleshooting**. If you have either `rubygems-bundler` or `open_gem` installed, please try removing them before filing an issue. For help with common problems, check out the [troubleshooting guide](doc/TROUBLESHOOTING.md).
+- **Reporting a bug**. If you’ve tried the troubleshooting guide and something is still not working, you can file an issue. Run `bundle-report-bug` to get all of the information you need to report a bug.
```
Comment (segiddins) [inline README.md]: what command is that?

File: README.md [hunk 4]
```diff
@@ -29,30 +35,31 @@ bundle install
...
-
-For help with common problems, see [TROUBLESHOOTING](doc/TROUBLESHOOTING.md).
-
-Still stuck? Try [filing an issue](doc/contributing/ISSUES.md).
-
-### Other questions
+### Contributing
 
-To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md).
+While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together. So if you'd like to contribute to Bundler, that's awesome, and we <3 you!
```
Comment (colby-swandale) [inline README.md]: I'm unsure about why we need this paragraph, can you explain why we need to talk about Ruby Together here?

File: README.md [hunk 5]
```diff
@@ -29,30 +35,31 @@ bundle install
...
 
-To get in touch with the Bundler core team and other Bundler users, please see [getting help](doc/contributing/GETTING_HELP.md).
+We have a [guide with recommended first steps](doc/contributing/README.md) that we suggest anyone interested in contributing to Bundler review first. Once you’ve completed those steps, feel free to get started contributing in any of the following ways:
```
Comment (colby-swandale) [inline README.md]: `feel free to get started`

I would suggest `feel free to start contributing`

File: README.md (hunk 3 above)
Comment (rubymorillo) [inline README.md]: @indirect can you confirm `bundle-report-bug` is the correct command? I remember seeing that in the doc we were both working off of

File: README.md (hunk 4 above)
Comment (rubymorillo) [inline README.md]: This graph already exists in the original README. I don't want to speak for @indirect but the way I read it is we're trying to be transparent about the fact that we do have paid contributors (not everyone might know that Bundler is a part of RT, or what RT is), but we also welcome and encourage volunteer contributions as well.

Comment (colby-swandale) [inline README.md]: Oh, i didn't realise this was already in the README. Nevermind then 👍

**Comment by rubymorillo** [comment]
@colby-swandale @segiddins I made two small changes per your feedback. Once @indirect tells us more about `bundle-report-bug` and the graph re: paid contributors through Ruby Together, I'll make those changes (won't be for a few days, though).

Thx!

**Comment by rubymorillo** [comment]
@indirect pinging about `bundle-report-bug` above; should I remove?

File: README.md [hunk 6]
```diff
@@ -10,15 +10,20 @@ Bundler makes sure Ruby applications run the same code on every machine.
 
 It does this by managing the gems that the application depends on. Given a list of gems, it can automatically download and install those gems, as well as any other gems needed by the gems that are listed. Before installing gems, it checks the versions of every gem to make sure that they are compatible, and can all be loaded at the same time. After the gems have been installed, Bundler can help you update some or all of them when new versions become available. Finally, it records the exact versions that have been installed, so that others can install the exact same gems.
 
+To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md), or [visit the Bundler site](http://bundler.io) to access the complete documentation.
```
Comment (olleolleolle) [inline README.md]: Perhaps `http://bundler.io ` => `https://bundler.io`

**Comment by colby-swandale** [comment]
ping @indirect

**Pull request closed without merging**